pom.xml

```diff
@@ -8,7 +8,8 @@
 
     <groupId>org.candlepin</groupId>
     <artifactId>candlepin-parent</artifactId>
-    <version>1.1</version>
+    <!-- This is the canonical version -->
+    <version>2.0</version>
```

Comment:
The version of "candlepin-parent".  It's not a version we use or a version that really matters much, but before the canonical location was in `buildfile`.  Now it's in `pom.xml`.